configstore/backends/dotenv.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+
+try:
+    import dotenv
+except ImportError:  # pragma: no cover
+    dotenv = None
+
+
+class DotenvBackend(object):
+
+    def __init__(self, dotenv_path):
+        if dotenv is None:
+            raise ImportError('install configstore[dotenv] to use '
+                              'the dotenv backend')
+
+        with open(dotenv_path) as file:
```

Comment:
The docker_secret backend could also check at init time that the directory exists.  Just saying.